target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -116,6 +116,11 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "1:lan" "2:lan" "3:lan" "4:lan" "5:wan"
 	;;
+	librerouter,librerouter-v1)
+		ucidef_set_interfaces_lan_wan "eth1.2" "eth0.1"
```

Comment:
```ucidef_set_interfaces_lan_wan "eth1.2" "eth0.1"```

Is this line still necessary? I was told some time ago that ucidef_add_switch will do the right thing here.

(Maybe the parameters for the switch0 just need to be reordered, Can you please check. Thanks)

Reply:
No, this line is not necesarry, thanks!